website/content/en/docs/config/mount/_index.md

```diff
@@ -12,7 +12,7 @@ The default mount type is shown in the following table:
 | < 0.10           | reverse-sshfs + Builtin SFTP server                           |
 | >= 0.10          | reverse-sshfs + OpenSSH SFTP server                           |
 | >= 0.17          | reverse-sshfs + OpenSSH SFTP server for QEMU, virtiofs for VZ |
-| >= 1.0 (Planned) | 9p for QEMU, virtiofs for VZ                                  |
+| >= 1.0           | 9p for QEMU, virtiofs for VZ                                  |
```

Comment:
You could shrink the width of the whole column now by 4 characters, if you want to.